src/user/UserRouter.ts

```diff
@@ -155,6 +155,14 @@ router.post(
         throw new HttpException(400, 'E-mail is not valid', errors.array());
       }
 
+      await UserService.passwordResetRequest(req.body.email);
```

Comment:
Will throw error if there's no user found with the e-mail attached in the req.body.email